docs/book/user-guide/advanced-guide/artifact-management/model-management.md

```diff
@@ -0,0 +1,267 @@
+
+```bash
+zenml model version update iris_logistic_regression --stage=...
+```
+
+### Promotion via Cloud Dashboard
+
+This feature is not yet available, but soon you will be able to promote your
+model versions directly from the ZenML Cloud dashboard. You can learn more about
+this feature [here](./model-control-plane-dashboard.md).
+
+### Promotion via Python SDK
+
+This is the most common way that you'll use to promote your model versions. You
+can see how you would do this here:
+
+```python
+from zenml import Client
+
+MODEL_NAME = "iris_logistic_regression"
+from zenml.enum import ModelStages
+
+client = Client()
+# get Model by name
+model = client.get_model(model_name=MODEL_NAME)
+
+# get specific Model Version and set it as Production
+# (if there is current Production version it will get Archived)
+model_version = model.get_version(version="1.2.3")
+model_version.set_stage(stage=ModelStages.PRODUCTION)
+
+# get Latest Model Version and set it as Staging
+# (if there is current Staging version it will get Archived)
+latest_model_version = model.versions[-1]
+latest_model_version.set_stage(stage=ModelStages.STAGING)
+```
+
+Within a pipeline context, you would get the model version from the step context
+but the mechanism for setting the stage is the same.
+
+```python
+from zenml import get_step_context, step, pipeline
+from zenml.enums import ModelStages
+
+@step
+def promote_to_staging():
+    model_config = get_step_context().model_config
+    model_version = model_config._get_model_version()
+    model_version.set_stage(ModelStages.STAGING, force=True)
+
+@pipeline(
+    ...
+)
+def train_and_promote_model():
+    ...
+    promote_to_staging(after=["train_and_evaluate"])
+```
+
+## Linking Artifacts to Models
+
+Artifacts generated during pipeline runs can be linked to models and specific model versions in ZenML. This connecting of artifacts provides lineage tracking and transparency into what data and models are used during training, evaluation, and inference.
+
+There are a few ways to link artifacts:
+
+### Configuring the Model
+
+The easiest way is to configure the `model_version` parameter on the `@pipeline` decorator or `@step` decorator:
+
+```python
+from zenml.model import ModelVersion 
+
+model_version = ModelVersion(
+    name="my_model",
+    version="1.0.0"
+)
+
+@pipeline(model_version=model_version)
+def my_pipeline():
+    ...
+```
+
+This will automatically link all artifacts from this pipeline run to the
+specified model version.
+
+### Artifact Configuration
+
+You can also explicitly specify the linkage on a per-artifact basis by passing
+special configuration to the Annotated output:
+
+```python
+@step
+def my_step() -> Annotated[
+    MyArtifact, 
+    "artifact_name",
+    DataArtifactConfig(model_name="my_model")
+]:
+   ...
+```
+
+The `DataArtifactConfig`, `ModelArtifactConfig`, and `EndpointArtifactConfig`
+types allow configuring model linkage directly on the artifact.
```

Comment:
I'm pretty sure that it is different now, @fa9r can you suggest a change here?

Reply:
Adapted it from [here](https://github.com/zenml-io/zenml/blob/439889f7dc2bf6230207870df783db0da8727fd6/docs/book/user-guide/starter-guide/track-ml-models.md), basically. I think the syntax is correct.